workspaces/src/network/config.rs

```diff
@@ -0,0 +1,156 @@
+//! Network specific configurations used to modify behavior inside a chain.
+//! This is so far only useable with sandbox networks since it would require
+//! direct access to a node to change the config. Each network like mainnet
+//! and testnet already have pre-configured settings; meanwhile sandbox can
+//! have additional settings on top of them to facilitate custom behavior
+//! such as sending large requests to the sandbox network.
+//
+// NOTE: nearcore has many, many configs which can easily change in the future
+// so this config.rs file just purely modifies the data and does not try to
+// replicate all the structs from nearcore side; which can be a huge maintenance
+// churn if we were to.
+
+use std::fs::File;
+use std::io::BufReader;
+use std::path::Path;
+
+use serde_json::{Map, Value};
+
+use crate::error::ErrorKind;
+use crate::Result;
+
+/// Replaces all entries recursively from `from` into `into`.
+fn overwrite_map(from: Map<String, Value>, mut into: Map<String, Value>) -> Result<Value> {
+    for (key, from_val) in from.into_iter() {
+        let Some(into_val) = into.remove(&key) else {
```

Comment:
This is unstable rust.
Do we require unstable to compile this crate?

Reply:
Are you talking about the destructing assignment? This was stabilized a year ago I think